how_to_document/README.md

```diff
@@ -0,0 +1,98 @@
+# How to ...  
+
+Here we describe best practices to document our project.  
+Please see below our collection of *"how to ...?"* questions, mostly linked to the use of git and github, 
+where we use plain English to explain how to accomplish certain documentation related tasks.
+
+Table of Contents
+=================
+* [How to start documenting a new project task (dataset, analysis)?](#how-to-start-documenting-a-new-project-task-dataset-analysis)
+* [How to write markdown files? <br>](#how-to-write-markdown-files-)
+* [How to break lines in markdown files?](#how-to-break-lines-in-markdown-files)
+* [How to get the changes made in github (new files/documentation) to your local repository?](#how-to-get-the-changes-made-in-github-new-filesdocumentation-to-your-local-repository)
+* [How to rename a folder or a file in a repository?](#how-to-rename-a-folder-or-a-file-in-a-repository)
+* [How to add a new file to the repository?](#how-to-add-a-new-file-to-the-repository)
+* [How to merge a branch into the master branch?](#how-to-merge-a-branch-into-the-master-branch)
+* [How to create a table of contents?](#how-to-create-a-table-of-contents)
+
+---------------
+
+## How to start documenting a new project task (dataset, analysis)?
+
+1. Go to the right github repository 
+3. Do a `git status` to double check your current branch
+4. if not in the *master* branch, go to it by using: `git checkout master`
+2. Do a `git pull` to make sure that you have the latest version of this repository
+5. Create a new (local) branch to start documenting a specific dataset and/or analysis: `git checkout -b NAMEOFBRANCH`   
+6. Synchronize the local and github repositories by: `git push -u origin NAMEOFBRANCH`   
+7. Create a folder -ideally with the name of the branch- and a README.md file in it  
+  * in github  
+     1. Open github in your browser, go to the repository and branch where you want to work  
+     2. Click on *create new file*   
+     3. In the "prompt" type your folder name then */* and then README.md  
```

Comment:
In the "prompt" type your folder name then */* and then README.md  

and then filename, for example README.md